examples/performance-testing-antipattern-examples.md

```diff
@@ -1,8 +1,14 @@
-# Performance Testing - Examples
+# Performance Testing - Antipattern Examples
 
 ## Antipattern 1: Ignoring Think Time
 
-Not including think time between user actions can lead to unrealistic performance metrics. In this example, the user actions are executed without any delay, which is unrealistic.
+Excluding think time between user actions can result in unrealistic performance
+metrics for certain types of tests, such as average, stress, and soak tests.
```

Comment:
I would like to see this in the glossary. Just a few words on what each type means.